### PR TITLE
added default.env, local.env.template + doku how to use it; dobi.sh d…

### PR DESCRIPTION
**Pull request by mlilien**
…ownloads dobi if not available; dobi.sh parameter checking; get rid of aliases; email notification in ci/cd on failure

### CODE REVIEW

File: dobi.sh [hunk 1]
```diff
@@ -1,20 +1,61 @@
...
-    dobi --filename meta.yaml list
+#check for docker
+if [[ $(docker version &>/dev/null;echo $?) -ne 0 ]]; then
+    echo "docker is not running"
     exit 1
 fi
 
-# load generated version infos
-source gen/gitversion/env/gitversion.env
-source gen/gitversion/branch/env/gitversion.env
```
Comment (JoergZeidler) [inline dobi.sh:17]: line is missing in new statement

File: README.md [hunk 2]
```diff
@@ -9,166 +9,107 @@ This building block integrates GitVersion ("GitVersion looks at your git history
...
 **Note: `credentials.yaml` is ignored by `.gitignore` and will not be checked in.**
 
 In further releases there will be a key value store to keep track of the users credentials.
 Before setting the pipeline you might login first to your concourse instance `fly -t <target> login --concourse-url http://<concourse>:<port>`. See the [fly documentation](https://concourse-ci.org/fly.html) for more help.
 Upload the pipeline file with fly:
 
-    $ fly -t <target> set-pipeline -n -p bb-buildingblock -l ci/config.yaml -l ci/credentials.yaml -c pipeline.yaml
+    $ fly -t <target> set-pipeline -n -p bb-gitversion -l ci/config.yaml -l ci/credentials.yaml -l ci/email.yaml -c pipeline.yaml
 
 After successfully uploading the pipeline to concourse CI login and unpause it. After that the pipeline should be triggered by new commits on the master branch (or new tags if enabled in `pipeline.yaml`).
```
Comment (JoergZeidler) [inline README.md]: email notification description is missing

File: dobi.sh (hunk 1 above)
Comment (mlilien) [reply dobi.sh:17]: thats a fix: we don't generate it anymore

File: README.md (hunk 2 above)
Comment (mlilien) [reply README.md]: done

Comment (mlilien) [reply README.md]: done again ;)

**Review by JoergZeidler — APPROVED** [review]

**Comment by mlilien** [comment]
also solves #17

**Pull request merged**